astropy/wcs/__init__.py

```diff
@@ -1,21 +1,24 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 """
 .. _wcslib: http://www.atnf.csiro.au/~mcalabre/WCS/
-.. _Paper IV: http://www.atnf.csiro.au/people/mcalabre/WCS/index.html
+.. _WCS papers: http://www.atnf.csiro.au/people/mcalabre/WCS/index.html
+.. _Paper IV: http://www.atnf.csiro.au/people/mcalabre/WCS/dcs_20040422.pdf
```

Comment:
I did notice the the "IV" was removed from the fourth paper on this page:

http://www.atnf.csiro.au/people/mcalabre/WCS/

But Mark Calabretta is still referring to it as Paper IV as recently as last week in wcslib:

https://github.com/mdboom/astropy/blob/wcs/upgrade-50/cextern/wcslib/C/dis.h#L31

I still think referring to it as the "FITS WCS distortion paper" is more clear than referring to it as "Paper IV", so I think I should make the change you suggest, but I am a little confused about the paper numbering at this point.